Sources/SwiftDocC/DocumentationService/Models/Services/Convert/ConvertRequest.swift

```diff
@@ -132,9 +132,9 @@ public struct ConvertRequest: Codable {
     
     /// The symbol identifiers that have an expanded documentation page available if they meet the associated access level requirement.
     ///
-    /// DocC sets the ``RenderMetadata/hasExpandedDocumentationForSymbols`` property to `true`
-    /// for these symbols if they meet the provided  requirements, so that renderers can display a "View More" link
-    /// that navigates the user to the full version of the documentation page.
+    /// For each of these symbols DocC sets the ``RenderMetadata/hasNoExpandedDocumentation`` property to `true`
```

Comment:
@daniel-grumberg does this updated description seem accurate to you?

Reply:
Yup that looks great!